esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino

```diff
@@ -42,6 +44,8 @@ for more details on the boards see mlrs-wifi-bridge-boards.h
 // Board
 // un-comment what you want
 //#define MODULE_GENERIC
+//#define MODULE_ESP32_DEV
```

Comment:
ok, another one: 
This is for the  NodeMCU ESP32-Wroom-32, so the flag should be called  MODULE_NODEMCU_ESP32_WROOM32

the idea of this pattern is to make it somewhat easy for a user to figure out which flag she/he needs to uncomment

Reply:
Done.